browser-test/src/question_lifecycle.test.ts

```diff
@@ -84,10 +82,18 @@ describe('normal question lifecycle', () => {
 
       await adminQuestions.expectActiveQuestions(allQuestions)
 
+      // Take screenshot of questions being published and active.
+      await adminQuestions.gotoAdminQuestionsPage()
+      await validateScreenshot(page)
```

Comment:
nit, blocking: Could we specify a custom file name here and below (e.g. address-active, address-draft)?

Right now, I feel like it's a bit hard to tell what the screenshot is testing by just looking at the file name.

Reply:
Although, I also see value in keeping as is since it's easier to trace it to the test that produced it. My personal preference would be to rename the screenshots, but up to you!

Reply:
Btw, https://github.com/civiform/civiform/pull/3233 will add the ability to set a custom name more easily!

Reply:
Thanks for refactoring. Updated screenshots.